README.md

```diff
@@ -40,22 +40,23 @@ If you have specific questions about the project, our [Telegram Group](https://t
 * Support for Bitcoin Mainnet & Testnet
 * Support for custom user-defined derivation paths
 * On-demand receive address verification
+* Single-sig and multi-sig change address verification
 * User-configurable QR code display density
+* Taproot (P2TR) single-sig signing
 * Responsive, event-driven user interface
+* Customized Linux live-boot OS to allow MicroSD card removal
 
 ### Considerations:
 * Built for compatibility with Specter Desktop, Sparrow, and BlueWallet Vaults
```

Comment:
Compatible with these Wallet co-ordinator softwares: Specter Desktop, Sparrow Desktop, BlueWallet, Nunchuk and Keeper.